### PR TITLE
F #3076: create one.vmpool.infoextended

### PR DESCRIPTION
**Pull request by al3xhh**
* Add new function dump_extended
    * Add new API call one.vmpool.infoextended
    * Add parameter --extended in the CLI

### CODE REVIEW

File: src/rm/RequestManagerPoolInfoFilter.cc [hunk 1]
```diff
@@ -615,6 +570,86 @@ void MarketPlacePoolInfo::request_execute(
...
         RequestAttributes& att)
 {
-    dump(att, ALL, -1, -1, "", "");
+    dump(att, ALL, -1, -1, "", "", false);
 }
 
+/* ------------------------------------------------------------------------- */
+/* ------------------------------------------------------------------------- */
+
+void RequestManagerPoolInfoFilter::pool_filter(
```
Comment (rsmontero) [inline src/rm/RequestManagerPoolInfoFilter.cc]: This Method should be for VirtualMachinePoolInfoFilter class

File: src/rm/RequestManagerPoolInfoFilter.cc [hunk 2]
```diff
@@ -476,7 +418,8 @@ void RequestManagerPoolInfoFilter::dump(
         int                start_id,
         int                end_id,
         const string&      and_clause,
-        const string&      or_clause)
+        const string&      or_clause,
+        bool               extended)
```
Comment (rsmontero) [inline src/rm/RequestManagerPoolInfoFilter.cc]: Make extended a class attribute so we do not need to pass it around. At the end is a property of the class info method e.g.  VirtualMachinePoolInfoExtended vs  VirtualMachinePoolInfo. The base class can have a bool exteneded attribute set in the class constructor to configure its properties.

**Comment by al3xhh** [comment]
Applies to master and 5.8

**Comment by al3xhh** [comment]
Pushed directly to master

**Pull request closed without merging**